### PR TITLE
Viz object. fixes #41

### LINKED ISSUE #41: Utilise the visualisation lib to draw a graph based upon storage
Somewhat an overlap with #3 however I wanted to make sure we had clarity here.

The extension will be made of the following parts:
1. Capture object
2. Storage object
3. Visualisation object
4. Lightbeam page code

Stages:
- **1.** stores into **2.**
- **3.** draws to a canvas
- **4.** reads from **2.** and draws with **3.** to a canvas in the page

### PR DESCRIPTION
**Pull request by princiya**
Issue #41 

Very basic setup to integrate capture and store with the viz object.

`lightbeam.js` calls `renderGraph()` by passing into it a `canvas context` and `websites object`.

`draw()` method in viz object takes in a `canvas context` and `websites object` and draws the graph.

**Fancy screenshot** ;)

screen shot 2017-06-14 at 00 13 36

### CODE REVIEW

File: js/capture.js [hunk 1]
```diff
@@ -52,4 +49,6 @@ const capture = {
       store.setFirstParty(documentUrl.hostname, data);
     }
   }
-};
\ No newline at end of file
+};
+
+capture.init();
```
Comment (princiya) [inline js/capture.js]: @biancadanforth moving `capture.init` after the object declaration.

File: js/store.js [hunk 2]
```diff
@@ -65,6 +67,14 @@ const store = {
 
   async remove() {
     return await browser.storage.local.remove('websites');
+  },
+
+  addListeners() {
+    browser.storage.onChanged.addListener((changes, area) => {
+      if (area === 'local' && changes.hasOwnProperty('websites')) {
+        viz.draw(changes['websites']['newValue']);
```
Comment (princiya) [inline js/store.js]: The call to render the graph needs to be re-written in a more efficient way.

File: js/viz.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line no-unused-vars
+const viz = {
+  _nodes: [],
+
+  draw(websites) {
```
Comment (princiya) [inline js/viz.js]: The idea is for this `draw()` function to call a preferred library.

File: index.html [hunk 4]
```diff
@@ -8,8 +8,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" href="images/lightbeam-48.png" type="image/x-icon">
     <link rel="icon" href="images/lightbeam-48.png" type="image/x-icon">
+    <link rel="stylesheet" href="css/style.css" type="text/css">
+    <script src="js/store.js" type="text/javascript"></script>
+    <script src="js/viz.js" type="text/javascript"></script>
```
Comment (princiya) [inline index.html]: Not sure if this is the right way to add script tags, inspite of adding these two files in `manifest`.
If I do not add them, `viz` object cannot get the reference to `getElementById` from this page to draw the graph.

File: js/viz.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    const svgElement = document.getElementById('canvas');
```
Comment (jonathanKingston) [inline js/viz.js]: This should probably be passed in perhaps. Not important for this PR though.

**Pull request merged**